### PR TITLE
Added history popup component and code for it

### PR DESCRIPTION
**Pull request by jacob-bro**
This will add the component needed for the history pop up window to function. Nothing is changed except to add the component file.
https://www.loom.com/share/c17e5924d0e44d829b53bb039ad6cbac

### CODE REVIEW

**Review by ChrisJRamirez — COMMENTED** [review]
For the most part the code looks good, I am just seeing a message in the review that says "React Hook useEffect has a missing dependency: 'options'. Either include it or remove the dependency array" and it is referencing line 27 on the client intake form. This may be out of the scope for this branch/task.

File: src/components/common/HistoryPopup.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
+import React, { Component } from "react";
+import "./styles.css";
+import ReactModal from "react-modal-resizable-draggable";
```
Comment (ashtilawat23) [inline src/components/common/HistoryPopup.js:3]: Any reason we couldn't use the ant-design model here? https://ant.design/components/modal/
Looks like you added a brand new dependency

File: src/components/common/HistoryPopup.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+// react-modal-resizable-draggable   0.1.6  is the dependency used here to make the pop up window funtion properly *SEE BELOW FOR NEEDED CSS
+
+class Popup extends Component {
```
Comment (ashtilawat23) [inline src/components/common/HistoryPopup.js:7]: Could you do a functional component instead? There are no other class components in the app

**Review by ashtilawat23 — CHANGES_REQUESTED** [review]
Could you convert this to a functional component? If we could move forward with the ant-design modal instead, that would be awesome! The `react-modal-resizable-draggle` hasn't been maintained or updated since 2021.

**Pull request merged**